packages/devtools-modules/client/shared/worker.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+export type Message = {
```

Comment:
Hmm - I agree, this is more than just unit tests. Is it okay to do in a separate commit?